dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/proxy/AbstractProxyFactory.java

```diff
@@ -110,7 +109,7 @@
     private <T> ClassLoader getClassLoader(Invoker<T> invoker) {
         ServiceModel serviceModel = invoker.getUrl().getServiceModel();
         ClassLoader classLoader = null;
-        if (serviceModel != null) {
+        if (serviceModel != null && serviceModel.getConfig() != null) {
             classLoader = serviceModel.getConfig().getInterfaceClassLoader();
```

Comment:
The `ProviderModel` used in `RestProtocolTest#registerProvider` does not specify the `ServiceConfig` value, which will cause the above NPE(`serviceModel.getConfig().getInterfaceClassLoader()`), so a judgment(`serviceModel.getConfig() != null`) is added